LibreNMS/Device/Sensor.php

```diff
@@ -400,7 +405,7 @@ protected static function processSensorData($sensors, $prefetch)
 
             // if no data set null and continue to the next sensor
             if (empty($data)) {
-                $data[$sensor['sensor_id']] = null;
+                $data[$sensor['sensor_id'] ?? 0] = null; // if no sensor_id, this is a new sensor, place at the first index
                 continue;
             }
```

Comment:
I don't get what this code even does, since `$data` is overwritten on each iteration.

Reply:
Pulled this into a separate PR.  I'll remove it from here.